index.js

```diff
@@ -12,6 +12,16 @@ function normalize(str) { // fix bug in v8
   return str.split('\u0000').map(function (s) { return s.normalize('NFC'); }).join('\u0000');
 }
 
+function containsNonASCII(str) {
```

Comment:
Regexes seem to be [much faster for this](https://jsperf.com/ascii-regex-vs-loop).